src/NuGet.Services.V3/Registration/Models/RegistrationPackageVulnerability.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Newtonsoft.Json;
+using NuGet.Protocol.Catalog;
+
+namespace NuGet.Protocol.Registration
+{
+    public class RegistrationPackageVulnerability : PackageVulnerability
+    {
+        [JsonProperty("registration", Order = 1)]
+        public string Registration { get; set; }
```

Comment:
Is this property used?

Reply:
Good catch--it was left over from copying existing types and deleting irrelevancies.